src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,8 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import React from 'react';
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
```

Comment:
You checked if `user` is defined in the `PostInfo` component so it's not necessary to add `?` before reading properties from the `user` object - FYI